streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1245,7 +1243,7 @@ protected StreamTask createStreamTask(final TaskId id, final Collection<TopicPar
                 time,
                 createProducer(id));
         } finally {
-            log.info("{} Created active task {} with assigned partitions {}", logPrefix, id, partitions);
+            log.trace("{} Created active task {} with assigned partitions {}", logPrefix, id, partitions);
```

Comment:
Note this is one log entry per task. On its caller (line 1341) we do have a debug-level entry:

```
 log.debug("{} Adding assigned standby tasks {}", logPrefix, partitionAssignor.standbyTasks());
```